tests/imgtype.go

```diff
@@ -79,60 +97,74 @@ func main() {
 
 		ref, err := is.Transport.ParseStoreReference(store, image)
 		if err != nil {
-			logrus.Fatalf("error parsing reference %q: %v", image, err)
-		}
-
-		src, err := ref.NewImageSource(systemContext, []string{expectedManifestType})
```

Comment:
The code for handling an error result from `ParseStoreReference`?  The diff is assuming one `if err != nil` is equivalent to another, but we should still be handling that case in the proposed version.